tests/migrations/test_operations.py

```diff
@@ -37,7 +37,8 @@ def test_create_table() -> None:
                 settings={"index_granularity": "256"},
             ),
         ).format_sql()
-        == "CREATE TABLE IF NOT EXISTS test_table (id String, name Nullable(String), version UInt64) ENGINE ReplacingMergeTree(version) ORDER BY version SETTINGS index_granularity=256;"
+        == "CREATE TABLE IF NOT EXISTS test_table (id String, name Nullable(String), version UInt64) ENGINE ReplicatedReplacingMergeTree(version) ORDER BY version SETTINGS index_granularity=256;"
+        or "CREATE TABLE IF NOT EXISTS test_table (id String, name Nullable(String), version UInt64) ENGINE ReplicatedReplacingMergeTree('/clickhouse/tables/events/{shard}/snuba_test/test_table', '{replica}', version) ORDER BY version SETTINGS index_granularity=256;"
```

Comment:
This isn't actually testing anything except the second condition on line 41 is not falsy, and won't fail even if the wrong SQL is produced

Reply:
Oops...